apps/site/app/[locale]/layout.tsx

```diff
@@ -15,8 +14,17 @@ import '@/styles/index.css';
 
 const fontClasses = classNames(IBM_PLEX_MONO.variable, OPEN_SANS.variable);
 
-const RootLayout: FC<PropsWithChildren> = async ({ children }) => {
-  const locale = await getLocale();
+type Props = {
+  params: Promise<{
+    locale: string;
+  }>;
+};
+
+const RootLayout: FC<PropsWithChildren<Props>> = async ({
+  children,
+  params,
+}) => {
+  const locale = (await params).locale;
```

Comment:
Can we just do await params and destruct?

const { locale } = await params;

Reply:
Sure!